.github/workflows/main.yml

```
@@ -21,75 +21,86 @@ jobs:
+  ubuntu-checks:
+    strategy:
+      matrix:
+        task: [build, test]
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v6
+        with:
+          persist-credentials: false
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          components: ${{ matrix.components }}
```

actionlint (1.7.12)

[error] property "components" is not defined in object type {task: string}

(expression)